### ISSUE TITLE
Does this work with zebra scanner ?

### ISSUE THREAD

**Issue by sumesh1993** [melihyarikkaya/react-native-serialport#9]
I am working on an application, that uses the zebra ds2208 scanner.
I am unable to read data though. i am using the sample code you have provided.
can u guide me in the right direction ?

**Comment by melihyarikkaya** [comment]
Can you connect?
Have you device's in the list  when `RNSerialport.getDeviceList()` is called ?

**Comment by sumesh1993** [comment]
Yes it shows up in the device list. Then i call `RNSerialport.connectDevice(deviceName, baudRate);`
But my function `onConnectedDevice()` is not called. What could be the issue ?
I am able to scan using[ this app](https://play.google.com/store/apps/details?id=com.zebra.scannercontrol.app)

**Comment by melihyarikkaya** [comment]
Did you do `DeviceEventEmitter.addListener(actions.ON_DEVICE_NOT_SUPPORTED, this.onUsbNotSupported, this)`? 
If not called `onUsbNotSupported()` on js,  see [this](https://www.zebra.com/content/dam/zebra_new_ia/en-us/manuals/barcode-scanners/ds2208-qsg-en.pdf) and [this](https://www.zebra.com/us/en/support-downloads/knowledge-articles/evm/emulating-a-com-serial-port-over-usb-using-cdc-driver.html) when on called `connectDevice()`

**Comment by melihyarikkaya** [comment]
[This application ](https://play.google.com/store/apps/details?id=com.zebra.scannercontrol.app) connects the device as a USB host.
Zebra has support for rs232.

You should use Rs232, but you may need to use the rs232 usb converter.
